src/software/ai/hl/stp/tactic/move_to_goal_line/BUILD

```
@@ -0,0 +1,48 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "move_to_goal_line_tactic",
+    srcs = ["move_to_goal_line_tactic.cpp"],
+    hdrs = [
+        "move_to_goal_line_fsm.h",
+        "move_to_goal_line_tactic.h",
+    ],
+    deps = [
+        "//shared:constants",
+        "//shared/parameter:cpp_configs",
+        "//software/ai/evaluation:enemy_threat",
+        "//software/ai/hl/stp/action:chip_action",
+        "//software/ai/hl/stp/action:move_action",
+        "//software/ai/hl/stp/tactic",
+        "//software/ai/hl/stp/tactic/chip:chip_tactic",
+        "//software/ai/hl/stp/tactic/dribble:dribble_tactic",
```

```suggestion
        "//software/ai/hl/stp/tactic",
        "//software/ai/hl/stp/tactic/move:move_tactic",
```
these dependencies aren't needed, but might need to update the includes to make it build